packages/core/logging/core-logging-browser-internal/src/appenders/console_appender.ts

```diff
@@ -8,6 +8,8 @@
 
 import type { Layout, LogRecord, DisposableAppender } from '@kbn/logging';
 
+// import { unsafeConsole } from '../../../../../../src/setup_node_env/harden/console';
```

Comment:
> ""non-package" code can not be imported from "common package" code."

I'm not sure the proper way to import `unsafeConsole` here.

Reply:
Ihmo the only way to have this code imported both by the hardening script (to patch) and by our logger (to retrieve the unmodified console) without the universe complaining about bad imports would be to move the content of what you did in `src/setup_node_env/harden/console.js` in a package